frontend/src/app/apps/[id]/page.tsx

```diff
@@ -1,104 +1,107 @@
-"use client";
+'use client';
 
-import React, { useEffect, useState } from "react";
-import styles from "@/styles/pages/application.module.css";
+import React, { useEffect, useState } from 'react';
+import styles from '@/styles/pages/application.module.css';
 import {
-  ApplicationExtended,
-  UserAppInformation,
-} from "@/app/api/applications/application.d";
-import ApplicationInformation from "@/components/ApplicationInformation/ApplicationInformation";
-import SkeletonLoader from "@/components/SkeletonLoader/SkeletonLoader";
-import BackLink from "@/components/BackLink/BackLink";
-import { useRouter } from "next/navigation";
+	ApplicationExtended,
+	UserAppInformation,
+} from '@/app/api/applications/application.d';
+import ApplicationInformation from '@/components/ApplicationInformation/ApplicationInformation';
+import SkeletonLoader from '@/components/SkeletonLoader/SkeletonLoader';
+import BackLink from '@/components/BackLink/BackLink';
+import { useRouter } from 'next/navigation';
 
 interface ApplicationProps {
-  params: {
-    id: string;
-  };
+	params: {
+		id: string;
+	};
 }
 
 const ApplicationPage = ({ params: { id } }: ApplicationProps) => {
-  const router = useRouter();
-  const [application, setApplication] = useState<ApplicationExtended>({
-    id: "",
-    name: "",
-    description: "",
-    owners: "",
-    teamName: "",
-    apiKey: "",
-    isValid: false,
-  });
-  const [loading, setLoading] = useState(true);
+	const router = useRouter();
+	const [application, setApplication] = useState<ApplicationExtended>({
+		id: '',
+		name: '',
+		description: '',
+		owners: '',
+		teamName: '',
+		apiKey: '',
```

Comment:
Could we institute some lint standards so we don't have so many unintentional changes in the PR?

Reply:
Sure: tabs, single quotes, semi colons, 80 print width